docs/man/maddy-filters.5.scd

```diff
@@ -566,6 +566,11 @@ Valid values:
 Allow multiple addresses in From header field for purposes of
 require_sender_match checks. Only first address will be checked, however.
 
+*Syntax*: sign_subdomains _boolean_ ++
+*Default*: no
+
+Allow to sign emails from subdomains using a top domain key.
```

Comment:
I think it would be useful to mention restriction to a one domain. How about...
```suggestion
Sign emails from subdomains using a top domain key.

Allows only one domain to be specified (can be workarounded using sign_dkim 
multiple times).
```

Reply:
Thanks for the edit.